routes/root.js

```diff
@@ -40,8 +40,16 @@ route.get('/logout', (req, res) => {
 })
 
 route.get('/leaderboard/:year?', (req, res) => {
-  const { year } = req.params
-  const options = {
+    let { year } = req.params
+    const validYears = [2020, 2019, 2018]
+  
+    if (!validYears.includes(year)) {
```

Comment:
Render 404 page if the year not present.

Reply:
updating

Reply:
rendering 404 page now.